refs/biblio.json

```diff
@@ -953,10 +953,98 @@
         "href": "https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-E.164-201011-I!!PDF-E&type=items",
         "date": "November 2010"
     },
+    "EBU-TT": {
+      "title": "EBU TECH 3350: \"EBU-TT Subtitling format definition\"",
+      "href": "https://tech.ebu.ch/docs/tech/tech3350.pdf",
+      "publisher": "European Broadcasting Union",
+      "versions": {
+        "1": {
+          "aliasOf": "EBU-TT-1-0"
+        },
+        "1.1": {
```

Comment:
I suspect the "." might be causing issues, here.

Why don't you put the actual content of the different versions in there instead, using the "1-1" notation?

Reply:
Thanks for the review, I noticed the tests weren't passing and have been learning/wrestling with a) working out how to locate which test is stalling and b) how to fix it, and came to the same conclusion! Fix upcoming shortly.

By the way, I worked my way to a way to find this, which you could consider adding to the documentation: if your `npm test` doesn't complete, try changing `package.json`'s `"test"` script command to execute `mocha` to add `-b` and change `-R min` to `-R list` so you can see where it hangs, and it bails on the first error.

Reply:
Yeah, I'm not too sure what happened, here, or why this stalled the tests. Usually other issues just throw and you get a readable message in the Travis log. Sorry about that.

Reply:
No worries @tobie , I've pushed an update which hopefully will address your comments and pass the CI tests (they do for me locally).

Reply:
Note I would still recommend using "1-1" here instead of "1.1" as the former's the currently accepted format and the latter is rather uncommon.

Reply:
Oh okay, I found at least one example with a `.` so I guessed that was acceptable. Will change.

Reply:
Pick whichever one you prefer.

Reply:
Just let me know either way so I can merge this. :)